packages/compass-sidebar/src/components/navigation-items.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useMemo } from 'react';
+import { connect } from 'react-redux';
+import {
+  useHoverState,
+  css,
+  cx,
+  ItemActionControls,
+  SmallIcon,
+  spacing,
+} from '@mongodb-js/compass-components';
+
+import type { ItemAction } from '@mongodb-js/compass-components';
+
+import DatabaseCollectionFilter from './database-collection-filter';
+import SidebarDatabasesNavigation from './sidebar-databases-navigation';
+
+import { changeFilterRegex } from '../modules/databases';
+
+type DatabasesActions = 'open-create-database';
+
+const navigationItem = css({
+  display: 'flex',
+  alignItems: 'center',
+  cursor: 'pointer',
+  color: 'var(--item-color)',
+  backgroundColor: 'var(--item-bg-color)',
+  height: spacing[5],
+  paddingLeft: spacing[3],
+  paddingRight: spacing[1],
+
+  ':hover': {
+    fontWeight: 'bold',
+  },
+});
+
+const activeNavigationItem = css({
+  color: 'var(--item-color-active)',
+  backgroundColor: 'var(--item-bg-color-active)',
+
+  // this is copied from leafygreen's own navigation, hence the pixel values
+  '::before': {
+    backgroundColor: 'var(--item-color-active)',
+    content: '""',
+    position: 'absolute',
+    left: '0px',
+    top: '6px',
+    bottom: '6px',
+    width: '4px',
+    borderRadius: '0px 6px 6px 0px',
```

Comment:
just a thought for the rebranding project, maybe worth seeing if we have many of these "rounded stuff" with duplicated values, we could extract a variable like "borderRadius" or something.

Reply:
Does that make sense, though? It is only for the focused sidebar indicator. Not a size that makes any sense in any other context or gets reused. Not like we'd ever change that variable. If/when the sidebar or branding changes again we'd probably replace this whole thing just like we're doing now.

Reply:
It is this vertical line on the left thingy, btw:

![Screenshot 2022-08-15 at 15 45 52](https://user-images.githubusercontent.com/69737/184657687-963625dc-512c-4d40-b159-e21381803517.png)

Reply:
Ahh ok yeah .. I guess then the radius is not meaningful